documentation/API_Release_Guidelines.md

```diff
@@ -4,59 +4,77 @@
+| maintenance release | A maintenance release is the release of a patch update of a public API version. |
+| release tag | A release tag is a GitHub tag placed on the main or a maintenance branch that identifies a release in the API's repository.|
+| release package | A release package is a zip file of the GitHub repository created using the GitHub release mechanism. It contains a snapshot of the full API repository marked with the release tag. |
+| GitHub release | A GitHub release is the combination of a release tag and, optionally, a release package of the GitHub repository (zip file) created using the GitHub release feature. A GitHub release applies to the full API repository. A GitHub release may containing any alpha, release-candidate or public API version(s). A GitHub release shall not include any wip API versions.`|
+| release PR | A release PR is created for an API version to prepare its GitHub release. A release PR shall minimally set the version and URL fields in the API yaml file to the exact API version and establish the availability of the API release assets as per the API readiness checklist. |
+| API release tracker | An API release tracker provides the visibility on the progress of the (pre-)releases of a given API version. Each public API version planned for release by an API Sub Project shall have its release tracker under their API Sub Project's API Release Tracking page. |
 
-(*) NOTE: all pre-releases are publicly available in the CAMARA GitHub and can be used AT THE USER'S OWN RISK, as changes may happen to such API versions without notice.
+(*) NOTE: pre-releases are not meant to be included in a meta-release. All pre-releases are publicly available in the CAMARA GitHub and can be used AT THE USER'S OWN RISK, as changes may happen to such API versions without notice.
 
-## API releases
+## API releases - overview
 
-In preparation of the public-release of an API version, an API Sub Project can create as many alpha and release-candidate API versions as needed for API development and testing. The API versioning is done as described in the API design guidelines (section on versioning). 
+### Release process
 
-The API Sub Project creates a release for an API version as follows:
+To prepare the release of a public API version, API versions shall be (pre-)released as follows:
 
-* a pre-release may be created for an alpha API version.
-* a pre-release must be created for each release-candidate API version.
-* a public-release must be created for the an API version in order to be published as part of a meta-release.
+* before M3, release one or more alpha API versions as needed
+* to reach M3, release the first release-candidate API version:
+  * the release-candidate implements the scope of the target public API version.
+  * this pre-release is agreed as ready for API implementation and functional testing.
+  * it is aligned with the release-candidate versions of Commonalities and ICM (M1).
```

Comment:
I thought the latest approach was, as for APIs, to have the (first) release-candidate for Comm. & ICM at M1 ? - Indeed the Wiki text is not up-to-date with that approach. Or do you want to keep the alpha release at M1 for Comm & ICM ?

Reply:
My proposal would be to keep the milestones for the current meta-release cycle (Fall24) and therefore for the first version of the release process as currently described within the wiki. The reasoning is that it is also how we are currently delivering the milestones.

That will give us the time to prepare the change for the next release cycle, aligning it first with the working groups (as it would mean that they have to deliver the first release candidate for the Spring25 release mid of October).

Reply:
so if we retain the pre-release of alpha API version of Comm & ICM for M1, then instead of changing the sentence to refer to M2 (in your above proposal), the sentence should be:
* it is aligned with the alpha version of Commonalities and ICM (M1). 
correct ?

Reply:
No, I thought about alignment with a release candidate of Commonalities and ICM. In this cycle the release candidates of Commonalities and ICM are late, so probably also the Sub Projects will have delays. Can we just omit the bracket for now and refer neither to M1 nor M2?